### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [tfeb/tfeb-lisp-hax#2]
Building with SBCL 2.2.7.28-02bc916fd / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id aed0708651ca4a7e08a0c7ddfa86d34cd1467473

*org.tfeb.hax/test* fails to build with the following error:

```
Unhandled SB-INT:SIMPLE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016C8003}>: Failed to find the TRUENAME of /home/quicklisp/quicklisp-controller/dist/build-cache/tfeb-lisp-hax/c338cd6f0fc3aedfcca35cbd77656413f2485b88/tfeb-lisp-hax-20220926-git/test/test-simple-loops.lisp: No such file or directory

```

[Full log here](http://report.quicklisp.org/2022-09-26/failure-report/tfeb-lisp-hax.html)

**Comment by tfeb** [comment]
Sorry.  Fixed by 3e22f2d *aka* v6.0.1

**Comment by tfeb** [comment]
Pretty sure this is fixed?
